### ISSUE TITLE
Missing MODEL_* constants on Vector3i

### ISSUE THREAD

**Issue by mournguard** [godotengine/godot#83751]
### Godot version

4.2.beta1

### System information

Godot v4.2.beta1 - Windows 10.0.22621 - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 3070 (NVIDIA; 31.0.15.3699) - 12th Gen Intel(R) Core(TM) i7-12700KF (20 Threads)

### Issue description

Commit [5fdc1232eff45e31ee53f58e618de6c58d3f7203](https://github.com/godotengine/godot/commit/5fdc1232eff45e31ee53f58e618de6c58d3f7203) added MODEL_* constants in an attempt to simplify Godot's front/back/forward/rear confusion but they've only been added to Vector3 and not Vector3i.

It would be a pretty simple fix but I don't have a setup to do it.

### Steps to reproduce

N/A

### Minimal reproduction project

N/A

**Comment by AThousandShips** [comment]
Feature proposals are tracked [here](https://github.com/godotengine/godot-proposals), this is for bugs. Please open a proposal instead.

**Comment by mournguard** [comment]
I wouldn't call this a proposal as it's very clearly an oversight, not a missing feature.

**Comment by AThousandShips** [comment]
I don't agree that it is an oversight, and I'm not sure the applicability of this to the integer vectors, hence why I think this is better served by discussing it on a proposal
